.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: c
```

Comment:
Does this file need to be in the root of the tree or can it be moved to the `.github` directory?

Reply:
Unfortunately it has to be in root https://github.com/travis-ci/travis-ci/issues/5519